ej2-angular/system-requirement.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: System requirements for Angular components | Syncfusion
-description: Learn here all about System requirement in Syncfusion Angular System requirement md component of Syncfusion Essential JS 2 and more.
+title: System requirements for Angular Platforms | Syncfusion
+description: Learn here all about system requirement support in Syncfusion Essential Angular platform, it's elements and more.
```

Comment:
Our primary keyword is angular components right, why you use angular platform? check and correct in all your changed places.

Reply:
still you did not change.